src/charactersheet/viewmodels/dm/encounter_sections/point_of_interest_section/index.html

```diff
@@ -182,6 +209,15 @@
             </small>
             <button type="submit"
                     class="btn btn-primary">Add</button>
+            <p class="text-muted text-left" data-bind="visible: showDisclaimer">
+              <sm><i>This data is distributed under the
+                <a href="http://media.wizards.com/2016/downloads/DND/SRD-OGL_V5.1.pdf"
+                  target="_blank">
+                  OGL</a><br/>
+                  Open	Game	License	v	1.0a	Copyright	2000,	Wizards	of	the	Coast,	LLC.
```

Comment:
kinda feels like a little extra to add the disclaimer for words that aren't really copyrighted? `wisdom`, `strength`, etc... I'm +0. just found it interesting

Reply:
I figured I might as well. No real reason for it.